paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -59,22 +57,18 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
     private val savedStateHandle: SavedStateHandle
 ) : ViewModel() {
 
-    val nameElement: SectionFieldElement = SimpleTextSpec.NAME.transform(
-        mapOf(
-            IdentifierSpec.Name to args.formArgs.billingDetails?.name
-        )
-    )
-    val name: StateFlow<String> = nameElement.getFormFieldValueFlow().map { formFieldsList ->
-        formFieldsList.firstOrNull()?.second?.takeIf { it.isComplete }?.value ?: ""
+    val nameController: TextFieldController = SimpleTextFieldController
+        .createNameSectionController(args.formArgs.billingDetails?.name)
+
+    val name: StateFlow<String> = nameController.formFieldValue.map { formFieldEntry ->
```

Comment:
I would recommend using the formFieldValue's isComplete field, or commenting why it is not used here.